zhaquirks/tuya/mcu/__init__.py

```diff
@@ -206,7 +221,19 @@ def from_cluster_data(self, data: TuyaClusterData) -> Optional[TuyaCommand]:
             datapoint_type = mapping.dp_type
             val = data.attr_value
             if mapping.dp_converter:
-                val = mapping.dp_converter(val)
+                args = []
+                if isinstance(mapping.attribute_name, tuple):
```

Comment:
It appears the tests aren’t hitting this case FYI

Reply:
They will, once I submit the PR for _TZE200_hkdl5fmv